packages/service-worker-mock/models/NotificationEvent.js

```diff
@@ -4,7 +4,8 @@ const ExtendableEvent = require('./ExtendableEvent');
 class NotificationEvent extends ExtendableEvent {
   constructor(args) {
```

Comment:
Looks like we were pretty off of the [spec](https://notifications.spec.whatwg.org/#service-worker-api). This should actually be 
```
constructor(type, init) {
  super(type, init);
  this.notification = init ? init.notification : null;
  this.action = init ? init.action : null;
}
```
Updating it would also require updating `utils/eventHandler.js` and maybe some tests.